protocol/directory.go

```diff
@@ -31,15 +36,18 @@ func InitDirectory(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey,
 	return d
 }
 
-func (d *ConiksDirectory) Update() {
+func (d *ConiksDirectory) Update(up *UpdatePolicies) {
```

Comment:
I think the `epochDeadline` should change every epoch (see https://github.com/coniks-sys/coniks-go/pull/80/commits/6b273ae7ac50dcda47091f950fa968bfd34793e7#r77198064). I'm wondering if the epoch deadline shouldn't be passed in from the server.